### PR TITLE
[J4.0] remove backslash from void tags

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue # .

### Summary of Changes

In HTML5, the backslash is not required on void elements, so this PR remove them from:

- ``
- `<br />`
- `<hr />`
- `<meta />`
- `<base />`
- `<link />`

### CODE REVIEW

File: administrator/components/com_admin/models/sysinfo.php [hunk 1]
```diff
@@ -388,7 +388,7 @@ public function &getPHPInfo()
 		preg_match_all('#<body[^>]*>(.*)</body>#siU', $phpInfo, $output);
 		$output = preg_replace('#<table[^>]*>#', '<table class="table table-striped adminlist">', $output[1][0]);
 		$output = preg_replace('#(\w),(\w)#', '\1, \2', $output);
-		$output = preg_replace('#<hr />#', '', $output);
```
Comment (wilsonge) [inline administrator/components/com_admin/models/sysinfo.php]: I think this is wrong - as this is parsing the ooutput of `phpinfo()`?? (unsure)

**Comment by RonakParmar** [comment]
Done code review and found "No newline at end of file".

administrator/language/en-GB/en-GB.com_weblinks.ini
administrator/language/en-GB/en-GB.plg_authentication_gmail.ini
administrator/language/en-GB/en-GB.plg_authentication_joomla.ini
administrator/language/en-GB/en-GB.plg_authentication_joomla.sys.ini
administrator/language/en-GB/en-GB.plg_system_sef.ini
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14000">issues.joomla.org/tracker/joomla-cms/14000</a>.</sub>

**Comment by C-Lodder** [comment]
@RonakParmar - Thanks. Fixed

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on c5b083c46317a2b425295edc09ae4e714bcae1db<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14000">issues.joomla.org/tracker/joomla-cms/14000</a>.</sub>

File: administrator/components/com_media/views/media/tmpl/default.php [hunk 2]
```diff
@@ -130,7 +130,7 @@
 		'footer' => '<a type="button" class="btn btn-secondary" data-dismiss="modal" aria-hidden="true">'
 			. JText::_('JLIB_HTML_BEHAVIOR_CLOSE') . '</a>'
 	),
-	'<div id="image" style="text-align:center;"><img id="imagePreviewSrc" src="../media/jui/img/alpha.png" alt="preview" style="max-width:100%; max-height:300px;"/></div>'
+	'<div id="image" style="text-align:center;"><img id="imagePreviewSrc" src="../media/jui/img/alpha.png" alt="preview" style="max-width:100%; max-height:300px;"></div>'
```
Comment (yvesh) [inline administrator/components/com_media/views/media/tmpl/default.php]: class="text-center"

**Comment by yvesh** [comment]
I have tested this item :white_check_mark: successfully on c5b083c46317a2b425295edc09ae4e714bcae1db<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14000">issues.joomla.org/tracker/joomla-cms/14000</a>.</sub>

File: administrator/components/com_media/views/media/tmpl/default.php (hunk 2 above)
Comment (C-Lodder) [inline administrator/components/com_media/views/media/tmpl/default.php]: will do in a separate PR

File: installation/language/ar-AA/ar-AA.ini [hunk 3]
```diff
@@ -132,7 +132,7 @@ INSTL_COMPLETE_LANGUAGE_DESC2="ملاحظة: جوملا! بحاجة للاتصا
 ; The word 'installation' should not be translated as it is a physical folder.
 INSTL_COMPLETE_REMOVE_FOLDER="يرجى إزالة مجلد التنصيب"
 ; The word 'installation' should not be translated as it is a physical folder.
-INSTL_COMPLETE_REMOVE_INSTALLATION="هام جداً... من أجل إكمال عملية التنصيب.<br/> يجب عليك القيام بإزالة مجلد التنصيب "_QQ_"installation"_QQ_" بشكل كامل. <br /> لن يمكنك الوصول إلى أبعد من هذه النقطة إلا بعد إزالة مجلد التنصيب. <br/>هذا من أجل حماية وأمان جوملا!"
+INSTL_COMPLETE_REMOVE_INSTALLATION="هام جداً... من أجل إكمال عملية التنصيب.<br/> يجب عليك القيام بإزالة مجلد التنصيب "_QQ_"installation"_QQ_" بشكل كامل. <br> لن يمكنك الوصول إلى أبعد من هذه النقطة إلا بعد إزالة مجلد التنصيب. <br/>هذا من أجل حماية وأمان جوملا!"
```
Comment (wilsonge) [inline installation/language/ar-AA/ar-AA.ini]: You missed one :P

File: libraries/vendor/phpmailer/phpmailer/src/Exception.php [hunk 4]
```diff
@@ -35,7 +35,7 @@ class Exception extends \Exception
      */
     public function errorMessage()
     {
-        $errorMsg = '<strong>' . $this->getMessage() . "</strong><br />\n";
+        $errorMsg = '<strong>' . $this->getMessage() . "</strong><br>\n";
```
Comment (wilsonge) [inline libraries/vendor/phpmailer/phpmailer/src/Exception.php]: Revert this one please. This is a 3rd party library

**Review by wilsonge — APPROVED** [review]

**Comment by wilsonge** [comment]
If you revert the phpmailer change I'm happy with this

**Comment by C-Lodder** [comment]
@wilsonge - installation languages done and phpmailer reverted

**Comment by wilsonge** [comment]
Nice job :)
